README.md

```diff
@@ -99,7 +99,34 @@ vars:
   pendo_source:
     pendo_<default_source_table_name>_identifier: your_table_name 
 ```
+If the above variable doesn't work, there's a workaround where you need to update your source.yml for pendo to use the correct table:
 
+```yml
+sources:
+  - name: pendo
+    database: <source_db>
+    
+    tables:
+      - name: <table_name>
+        identifier: >
+          {% if target.type == 'snowflake' %}{{ var('pendo_<default_source_table_name>_identifier', '"<New Name>"') }}
+          {% else %}{{ var('pendo_<default_source_table_name>_identifier', '<table_name>') }}{% endif %}       
+```
+For Example: 
+
+```yml
+version: 2
+
+sources:
+  - name: pendo
+    database: edw_common_stg_db
+    
+    tables:
+      - name: group
+        identifier: >
+          {% if target.type == 'snowflake' %}{{ var('pendo_group_identifier', '"GROUP"') }}
+          {% else %}{{ var('pendo_group_identifier', 'group') }}{% endif %}        
```

Comment:
Hi @RichardThRivera! Thank you for this suggestion. Would you be able to incorporate this suggestion into the `models/src_pendo.yml` in this PR? If not no worries, I'll add it in towards the end of the week.